sdk/servicebus/microsoft-azure-servicebus/src/test/java/com/microsoft/azure/servicebus/ManagementClientProxyTest.java

```diff
@@ -0,0 +1,57 @@
+package com.microsoft.azure.servicebus;
+
+import com.microsoft.azure.servicebus.management.ManagementClient;
+import com.microsoft.azure.servicebus.management.QueueDescription;
+import com.microsoft.azure.servicebus.primitives.ServiceBusException;
+import org.junit.Assert;
+import org.junit.Test;
+import java.io.IOException;
+import java.net.*;
+import java.util.*;
+import java.net.InetSocketAddress;
+import java.net.ProxySelector;
+import java.net.URI;
+import java.util.LinkedList;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+
+public class ManagementClientProxyTest {
+    @Test
+    public void managementClientWithProxy() throws InterruptedException, ServiceBusException {
```

Comment:
>managementClientWithProxy [](start = 16, length = 25)

You can ignore this test. The CI/CD machine doens't have a proxy server and hence this won't run there.